pkg/websocket/client.go

```diff
@@ -503,7 +519,9 @@ const (
 
 	defaultReconnectInterval = 60 * time.Second
 
-	defaultWriteWait = 10 * time.Second
+	defaultCloseDelayPeriod = 10 * time.Second
```

Comment:
Sorry just caught this – this seems like a long time to sleep for. Given that most writes across a network should complete within 1 second even on a slow connection, it seems safe to reduce this number as well in order to ensure a speedy close sequence while still giving ample time for the server to receive the close frame. Thoughts?

Reply:
Works for me! Updated